sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -51,6 +51,7 @@
 from apache_beam.runners.dataflow.internal import names
 from apache_beam.runners.dataflow.internal.clients import dataflow
 from apache_beam.runners.dataflow.internal.names import PropertyNames
+from apache_beam.runners.internal.names import BEAM_PACKAGE_NAME, BEAM_SDK_NAME
```

Comment:
I believe it would be more clear, if you import names, and then use names.BEAM_PACKAGE_NAME etc. in the code. What do you think?

Reply:
Sounds good. I had to import as `shared_names` to avoid conflict with the import on line 51 though.